### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-style` -> `custom-style`
`mixin` -> `@apply`
`version++`
`polymer++`
`@license` tags

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-core-icon
+iron-icon
 =========
 
 See the [component page](http://polymer-project.org/docs/elements/core-elements.html#core-icon) for more information.
```
Comment (cdata) [inline README.md]: There are more references to `core-` on this page :cry:

File: demo/index.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <!--
+@license
 Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
```
Comment (cdata) [inline demo/index.html]: :two::zero::one::five:

File: iron-icon.html [hunk 3]
```diff
@@ -57,11 +63,7 @@
...
 -->
 
-<link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="../iron-meta/iron-meta.html">
-<link rel="import" href="../iron-flex-layout/iron-flex-layout.html">
-
-<style is="x-style">
+<style is="custom-style">
   * {
     --iron-icon-size: 24px;
```
Comment (cdata) [inline iron-icon.html:68]: This should be changed into a default value below.

File: README.md (hunk 1 above)
Comment (notwaldorf) [inline README.md]: No good deed ever goes unpunished.

**Comment by notwaldorf** [comment]
Done! Punting on the default values for now.

**Comment by cdata** [comment]
:+1: punishment complete.

**Comment by notwaldorf** [comment]
:tada: :fireworks: :confetti_ball:

**Pull request merged**